package.json

```diff
@@ -104,6 +104,7 @@
     "jest": "^27.4.5",
     "jest-cli": "^27.4.5",
     "jest-environment-node": "^27.4.4",
+    "jquery": "https://github.com/jquery/jquery/tarball/c98597eaf5e144ee5e549cb41984687cd1033068",
```

Comment:
For my knowledge, can you explain what this is pulling and why we're pulling direct from Github?